test/functional/p2p_feefilter.py

```diff
@@ -61,7 +55,9 @@ def set_test_params(self):
         # mempool and wallet feerate calculation based on GetFee
         # rounding down 3 places, leading to stranded transactions.
         # See issue #16499
-        self.extra_args = [["-minrelaytxfee=0.00000100", "-mintxfee=0.00000100"]] * self.num_nodes
+        # grant noban permission to all peers to speed up tx relay / mempool sync
+        self.extra_args = [["-minrelaytxfee=0.00000100", "-mintxfee=0.00000100",
+                            "-whitelist=noban@127.0.0.1"]] * self.num_nodes
```

Comment:
would be nice to either put all in one line or each in one new line

Reply:
Decided for the one-arg-per-line version. Done.